### DISCUSSION TITLE
使用Xray配置tap虚拟网卡遇到的问题

### DISCUSSION THREAD

**Discussion by morckclient** [XTLS/Xray-core#355] [Q&A]
首先说明，这不是Xray的问题，我在寻求这个问题的解决办法，希望有人能帮帮我。
配置Xray的socks端口为10801
使用tun2socks配置tap虚拟网卡并连接 `socks://127.0.0.1:10801` 

配置socks服务器为本地127.0.0.1时，怎么设置系统的默认路由？求指教

删除默认路由    `route delete 0.0.0.0 mask 0.0.0.0`
添加虚拟网卡出口    `route add 0.0.0.0 mask 0.0.0.0 10.0.0.1 metric 6`
代理转发网关    `route add 127.0.0.1 192.168.1.1 metric 5`

这么添加似乎无效...

然而我用远程socks服务器地址这么配置却可以用。

**Comment by KyonCN** [comment]
如果这不是xray的问题 你应该发到discussion或者去tg群

**Comment by ghost** [comment]
可能需要配置 sendThrough？
https://xtls.github.io/config/base/outbounds/
